README.md

```diff
@@ -1,12 +1,17 @@
 ![PODMAN logo](https://raw.githubusercontent.com/containers/common/main/logos/podman-logo-full-vert.png)
 
 # Podman: A tool for managing OCI containers and pods
+![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/containers/podman)
+[![Go Report Card](https://goreportcard.com/badge/github.com/containers/libpod)](https://goreportcard.com/report/github.com/containers/libpod)
+
+<br/>
 
 Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers.
 Podman runs containers on Linux, but can also be used on Mac and Windows systems using a Podman-managed virtual machine.
 Podman is based on libpod, a library for container lifecycle management that is also contained in this repository. The libpod library provides APIs for managing containers, pods, container images, and volumes.
 
-* [Latest Version: 4.4.0](https://github.com/containers/podman/releases/tag/v4.4.0)
+* [Latest Version: 4.4.4](https://github.com/containers/podman/releases/tag/v4.4.4)
```

Comment:
IDK if it's at all possible, but it would be great if we could get this from the "Latest" release tag in the release section.

Reply:
It is possible to retrieve the latest release via github api but I dont think there is a way to do this in the readme (except for the already added badge).

In this example the call will be a GET to the following url: *https://api.github.com/repos/containers/podman/releases/latest*
and the result will be:
```json
{
  "url": "https://api.github.com/repos/containers/podman/releases/99564686",
  "assets_url": "https://api.github.com/repos/containers/podman/releases/99564686/assets",
  "upload_url": "https://uploads.github.com/repos/containers/podman/releases/99564686/assets{?name,label}",
  "html_url": "https://github.com/containers/podman/releases/tag/v4.5.0",
  "id": 99564686,
  "author": {
    "login": "lsm5",
    "id": 247367,
    "node_id": "MDQ6VXNlcjI0NzM2Nw==",
    "avatar_url": "https://avatars.githubusercontent.com/u/247367?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/lsm5",
    "html_url": "https://github.com/lsm5",
    "followers_url": "https://api.github.com/users/lsm5/followers",
    "following_url": "https://api.github.com/users/lsm5/following{/other_user}",
    "gists_url": "https://api.github.com/users/lsm5/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/lsm5/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/lsm5/subscriptions",
    "organizations_url": "https://api.github.com/users/lsm5/orgs",
    "repos_url": "https://api.github.com/users/lsm5/repos",
    "events_url": "https://api.github.com/users/lsm5/events{/privacy}",
    "received_events_url": "https://api.github.com/users/lsm5/received_events",
    "type": "User",
    "site_admin": false
  },
  "node_id": "RE_kwDOBoFt0c4F7zyO",
  "tag_name": "v4.5.0",
  "target_commitish": "main",
  "name": "v4.5.0",
  "draft": false,
  "prerelease": false,
  "created_at": "2023-04-14T15:08:44Z",
  "published_at": "2023-04-14T15:23:29Z",
  "assets": [

  ],
  "tarball_url": "https://api.github.com/repos/containers/podman/tarball/v4.5.0",
  "zipball_url": "https://api.github.com/repos/containers/podman/zipball/v4.5.0",
  "body": "### Features\r\n- The `podman kube play` command now supports the hostIPC field ([#17157](https://github.com/containers/podman/issues/17157)).\r\n- The `podman kube play` command now supports a new flag, `--wait`, that keeps the workload running in foreground until killed with a sigkill or sigterm. The workloads are cleaned up and removed when killed ([#14522](https://github.com/containers/podman/issues/14522)).\r\n- The `podman kube generate` and `podman kube play` commands now support SELinux filetype labels.\r\n- The `podman kube play` command now supports sysctl options ([#16711](https://github.com/containers/podman/issues/16711)).\r\n- The `podman kube generate` command now supports generating the Deployments ([#17712](https://github.com/containers/podman/issues/17712)).\r\n- The `podman machine inspect` command now shows information about named pipe addresses on Windows ([#16860](https://github.com/containers/podman/issues/16860)).\r\n- The `--userns=keep-id` option for `podman create`, ` run`, and `kube play` now works for root containers by copying the current mapping into a new user namespace ([#17337](https://github.com/containers/podman/issues/17337)).\r\n- A new command has been added, `podman secret exists`, to verify if a secret with the given name exists.\r\n- The `podman kube generate` and `podman kube play` commands now support ulimit annotations ([#16404](https://github.com/containers/podman/issues/16404)).\r\n- The `podman create`, `run`, `pod create`, and `pod clone` commands now support a new option, `--shm-size-systemd`, that allows limiting tmpfs sizes for systemd-specific mounts ([#17037](https://github.com/containers/podman/issues/17037)).\r\n- The `podman create` and `run` commands now support a new option, `--group-entry` which customizes the entry that is written to the `/etc/group` file within the container when the `--user` option is used ([#14965](https://github.com/containers/podman/issues/14965)).\r\n- The `podman create` and `podman run` commands now support a new option, `--security-opt label=nested`, which allows SELinux labeling within a confined container.\r\n- A new command, `podman machine os apply` has been added, which applies OS changes to a Podman machine, from an OCI image.\r\n- The `podman search` command now supports two new options: `--cert-dir` and `--creds`.\r\n- Defaults for the `--cgroup-config` option for `podman create` and `podman run` can now be set in `containers.conf`.\r\n- Podman now supports auto updates for containers running inside a pod ([#17181](https://github.com/containers/podman/issues/17181)).\r\n- Podman can now use a SQLite database as a backend for increased stability. The default remains the old database, BoltDB. The database to use is selected through the `database_backend` field in `containers.conf`.\r\n- Netavark plugin support has been added. The netavark network backend now allows users to create custom network drivers. `podman network create -d <plugin>` can be used to create a network config for your plugin and then Podman will use it like any other config and takes care of setup/teardown on container start/stop. This requires at least Netavark version 1.6.\r\n- DHCP with macvlan and the netavark backend is now supported.\r\n\r\n### Changes\r\n- Remote builds using the `podman build` command no longer allows `.containerignore` or `.dockerignore` files to be symlinks outside the build context.\r\n- The `podman system reset` command now clears build caches.\r\n- The `podman play kube` command now adds ctrName as an alias to the pod network ([#16544](https://github.com/containers/podman/issues/16544)).\r\n- The `podman kube generate` command no longer adds hostPort to the pod spec when generating service kinds.\r\n- Using a private cgroup namespace with systemd containers on a cgroups v1 system will explicitly error (this configuration has never worked) ([#17727](https://github.com/containers/podman/issues/17727)).\r\n- The `SYS_CHROOT` capability has been re-added to the default set of capabilities.\r\n- Listing large quantities of images with the `podman images` command has seen a significant performance improvement ([#17828](https://github.com/containers/podman/issues/17828)).\r\n\r\n### Quadlet\r\n- Quadlet now supports the `Rootfs=` option, allowing containers to be based on rootfs in addition to image.\r\n- Quadlet now supports the Secret key in the Container group.\r\n- Quadlet now supports the Logdriver key in `.container` and `.kube` units.\r\n- Quadlet now supports the Mount key in `.container` files ([#17632](https://github.com/containers/podman/issues/17632)).\r\n- Quadlet now supports specifying static IPv4 and IPv6 addresses in `.container` files via the IP= and IP6= options.\r\n- Quadlet now supports health check configuration in `.container` files.\r\n- Quadlet now supports relative paths in the Volume key in .container files ([#17418](https://github.com/containers/podman/issues/17418)).\r\n- Quadlet now supports setting the UID and GID options for `--userns=keep-id` ([#17908](https://github.com/containers/podman/issues/17908)).\r\n- Quadlet now supports adding `tmpfs` filesystems through the `Tmpfs` key in `.container` files ([#17907](https://github.com/containers/podman/issues/17907)).\r\n- Quadlet now supports the `UserNS` option in `.container` files, which will replace the existing `RemapGid`, `RemapUid`, `RemapUidSize` and `RemapUsers` options in a future release ([#17984](https://github.com/containers/podman/issues/17984)).\r\n- Quadlet now includes a `--version` option.\r\n- Quadlet now forbids specifying SELinux label types, including disabling selinux separation.\r\n- Quadlet now does not set log-driver by default.\r\n- Fixed a bug where Quadlet did not recognize paths starting with systemd specifiers as absolute ([#17906](https://github.com/containers/podman/issues/17906)).\r\n\r\n### Bugfixes\r\n- Fixed a bug in the network list API where a race condition would cause the list to fail if a container had just been removed ([#17341](https://github.com/containers/podman/issues/17341)).\r\n- Fixed a bug in the `podman image scp` command to correctly use identity settings.\r\n- Fixed a bug in the remote Podman client's `podman build` command where building from stdin would fail.  `podman --remote build -f -` now works correctly ([#17495](https://github.com/containers/podman/issues/17495)).\r\n- Fixed a bug in the `podman volume prune` command where exclusive (`!=`) filters would fail ([#17051](https://github.com/containers/podman/issues/17051)).\r\n- Fixed a bug in the `--volume` option in the `podman create`, `run`, `pod create`, and `pod clone` commands where specifying relative mappings or idmapped mounts would fail ([#17517](https://github.com/containers/podman/issues/17517)).\r\n- Fixed a bug in the `podman kube play` command where a secret would be created, but nothing would be printed on the terminal ([#17071](https://github.com/containers/podman/issues/17071)).\r\n- Fixed a bug in the `podman kube down` command where secrets were not removed.\r\n- Fixed a bug where cleaning up after an exited container could segfault on non-Linux operating systems.\r\n- Fixed a bug where the `podman inspect` command did not properly list the network configuration of containers created with `--net=none` or `--net=host` ([#17385](https://github.com/containers/podman/issues/17385)).\r\n- Fixed a bug where containers created with user-specified SELinux labels that created anonymous or named volumes would create those volumes with incorrect labels.\r\n- Fixed a bug where the `podman checkpoint restore` command could panic.\r\n- Fixed a bug in the `podman events` command where events could be returned more than once after a log file rotation ([#17665](https://github.com/containers/podman/issues/17665)).\r\n- Fixed a bug where errors from systemd when restarting units during a `podman auto-update` command were not reported.\r\n- Fixed a bug where containers created with the `--health-on-failure=restart` option were not restarting when the health state turned unhealthy ([#17777](https://github.com/containers/podman/issues/17777)).\r\n- Fixed a bug where containers using the `slirp4netns` network mode with the `cidr` option and a custom user namespace did not set proper DNS IPs in `resolv.conf`.\r\n- Fixed a bug where the `podman auto-update` command could fail to restart systemd units ([#17607](https://github.com/containers/podman/issues/17607)).\r\n- Fixed a bug where the `podman play kube` command did not properly handle `secret.items` in volumes ([#17829](https://github.com/containers/podman/issues/17829)).\r\n- Fixed a bug where the `podman generate kube` command could generate pods with invalid names and hostnames ([#18054](https://github.com/containers/podman/issues/18054)).\r\n- Fixed a bug where names of limits (such as `RLIMIT_NOFILE`) passed to the `--ulimit` option to `podman create` and `podman run` were case-sensitive ([#18077](https://github.com/containers/podman/issues/18077)).\r\n- Fixed a possible corruption issue with the configuration state of `podman machine` during system failures on Mac, Linux, and Windows.\r\n\r\n### API\r\n- The Compat Stats endpoint for Containers now returns the `Id` key as lowercase `id` to match Docker ([#17869](https://github.com/containers/podman/issues/17869)).\r\n- Fixed a bug where the Compat top endpoint incorrectly returned titles as a string instead of a list ([#17524](https://github.com/containers/podman/issues/17524)).\r\n\r\n### Misc\r\n- The `podman version` command no longer joins the rootless user namespace ([#17657](https://github.com/containers/podman/issues/17657)).\r\n- The `podman-events --stream` option is no longer hidden and is now documented.\r\n- Updated Buildah to v1.30.0\r\n- Updated the containers/storage library to v1.46.1\r\n- Updated the containers/image library to v5.25.0\r\n- Updated the containers/common library to v0.52.0"
}
```

My suggestion would be to remove the hand written version and just show the badge.

Reply:
@R3DRUN3 please make the change in the PR. LGTM after that.

Reply:
@lsm5 added a commit to the PR with the required changes 👍